docs/avdanos/contributing/design/OXYDE/fourth-style.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 5
+title: Animation
+description: Making things feel more alive.
+---
+
+:::caution
+Reminder: OXYDE stills in Beta phase.
+:::

```

Comment:
You've done this numerous times — so see this comment for the others.

SPaG: ~~stills~~ _still_

**Use the WIP snippet.** The WIP snippet links to a document's source code on GitHub such that they can contribute. It is used in other documentation.